package/gluon-web-admin/luasrc/lib/gluon/web/model/admin/remote.lua

```diff
@@ -30,11 +31,17 @@ function keys:write(value)
 	end
 end
 
+local min_password_length = ((site.config_mode or {}).remote_login or {}).min_password_length or 12
```

Comment:
Usually I would agree, but here we try to fix an existing serious security whole. So adding a minimum length is cruicial for the security of the nodes. I cannot think of a case, where a short password is needed.

Also here there is no drawback for users, that used a short password before, because there cannot be no check of en existing password that is too short.